src/color-chart/color-chart-global.css

```diff
@@ -40,9 +40,12 @@ color-chart > color-scale {
 	}
 
 	&::part(color-swatch)::before {
-		content: "Hi";
+		content: var(--label);
```

Comment:
Don't think we really want a label, given the usual density of this kind of data, but I suppose it can't hurt? Def not a priority though, especially before we get the basic stuff working…

Reply:
I was planning to use that to draw the line